components/portal/messages/controllers.js

```diff
@@ -86,16 +86,15 @@ define(['angular'], function(angular) {
                 messagesService.getMessagesByTitle(allMessages),
             };
           }
-            // Call filtered notifications promises, then pass on to
-            // the completion function
-            $q.all(promiseFilteredMessages)
-              .then(filterMessagesSuccess)
-              .catch(filterMessagesFailure);
-         
-            // Separate all messages by their types
-            $scope.messages = $filter('separateMessageTypes')(allMessages);
-            // Change hasMessages so child controllers can pick up on it
-            $scope.hasMessages = true;
+          // Call filtered notifications promises, then pass on to
+          // the completion function
+          $q.all(promiseFilteredMessages)
```

Comment:
This works :white_check_mark: 
But this is an area where `$q` diverged from `Promise`. In the `Promise` spec, `all` only supports iterable. [ref 1] Plain Object is not considered an interable. [ref 2] Should there be a plan to upgrade to ES2015+ this would be an area to flag as a risky upgrade point.

[ref 1]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/all
[ref 2]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Iteration_protocols#The_iterable_protocol